dist/tools/randhex/randhex.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+bits="${1:-32}"
```

Comment:
this can be replaced by `: ${bits:=32}` to work most non-bash `/bin/sh`

Reply:
The `:-` syntax is POSIX compliant.

From [The Open Group Base Specifications](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02):

> ${parameter:-word}: Use Default Values. If parameter is unset or null, the expansion of word shall be substituted; otherwise, the value of parameter shall be substituted.

Reply:
hm. I stand corrected and confused.